### PR TITLE
Add Log statement about statsd receiver

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 
This PR is created to add log statement regarding statsd receiver

Tested: 

```
2023/08/07 21:35:37 found no extra config, skip it, err: open /opt/aws/aws-otel-collector/etc/extracfg.txt: no such file or directory
2023/08/07 21:35:37 attn: users of the statsd please refer to https://github.com/aws-observability/aws-otel-collector/issues/2249 in regards to an ADOT Collector v0.32.0 breaking change
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cmd/awscollector/main.go [hunk 1]
```diff
@@ -93,6 +93,10 @@ func main() {
 func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error) {
 	flagSet := config.Flags(featgate)
 
+	// TODO: remove after ADOT Collector v0.34.0 is released
+	log.Printf("attn: users of the statsd please refer to " +
```
Comment (humivo) [inline cmd/awscollector/main.go:97]: nit: do we want to specify the `stasd receiver` here?

Comment (vasireddy99) [reply cmd/awscollector/main.go:97]: Yes, the warning issue linked is about the statsd receiver.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ Use the community resources below for getting help with the ADOT Collector.
 * If you think you may have found a bug, open a [bug report](https://github.com/aws-observability/aws-otel-collector/issues/new?template=bug_report.md).
 * For contributing guidelines, refer to [CONTRIBUTING.md](CONTRIBUTING.md).
 
+### Notice: ADOT Collector v0.32.0 Breaking Change
```
Comment (bryan-aguilar) [inline README.md:21]: There is no breaking change in v0.32.0. The breaking change could be introduced in v0.33.0 at the earliest.

File: cmd/awscollector/main.go (hunk 1 above)
Comment (humivo) [reply cmd/awscollector/main.go:97]: Oh I meant the wording here like this
```suggestion
	log.Printf("attn: users of the statsd receiver please refer to " +
```

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**